.github/workflows/main.yml

```diff
@@ -31,6 +31,9 @@ jobs:
             -   name: "PHPStan"
                 run: ./bin/phpstan analyze -l 8 src examples tests/GenericCases
```

Comment:
Maybe we should remove `GenericCases` from here?

Reply:
Removed. I've moved `examples` to level 9 too. 

Unfortunately, main `src` directory must stay on level 8, because on 9 there are too many false positives related to generics in plain strings array

Reply:
Like this:

```
 ------ -------------------------------------------------------------------------------------------------------------------------------------------------------------------- 
  Line   src/PlainString.php                                                                                                                                                 
 ------ -------------------------------------------------------------------------------------------------------------------------------------------------------------------- 
  274    Parameter #1 $strings of class GW\Value\PlainStringsArray constructor expects GW\Value\ArrayValue<GW\Value\StringValue|string>, GW\Value\ArrayValue<string> given.  
  294    Parameter #1 $strings of class GW\Value\PlainStringsArray constructor expects GW\Value\ArrayValue<GW\Value\StringValue|string>, GW\Value\ArrayValue<string> given.  
 ------ -------------------------------------------------------------------------------------------------------------------------------------------------------------------- 
```